haskell-src/Concordium/Types/Queries.hs

```diff
@@ -63,7 +63,7 @@ data ConsensusStatus = ConsensusStatus
       -- |Time of the (original) genesis block
       csGenesisTime :: !UTCTime,
       -- |(Current) slot duration in milliseconds
-      csSlotDuration :: !Duration,
+      csSlotDuration :: !(Maybe Duration),
```

Comment:
Can you please update documentation here as well to reflect this is present if and only if protocol version is <= 5.